beast/examples/phat_small/regress_check.py

```diff
@@ -36,8 +37,8 @@ def hdf5diff(fname1, fname2):
     hdfb = h5py.File(fname2, 'r')
 
     hd = hdf5diff_results()
-    for sname in hdfa.keys():
-        if sname not in hdfb.keys():
+    for sname in list(hdfa.keys()):
```

Comment:
list() is not needed here. a for-loop does the job, unless h5py sucks

Reply:
Fixed.  h5py doesn't seem to suck.  :-)